### PR TITLE
Update webcam while recording

### LINKED ISSUE #101: Can't see webcam while recording a testimonial
## Description
When recording a video, the user doesn't know that they are being recorded or what is being recorded because all they see is a black screen (pictured below) which is the video element with no `src` attribute.

Screen Shot 2021-02-04 at 4 02 43 PM

## Suggestion
Add a webcam component if they haven't recorded a video yet so the user is able to see themself and replace the webcam component with the video element once the recording is finished.

### Possible libraries 
- [vue-web-cam](https://github.com/vinceg/vue-web-cam)

### PR DESCRIPTION
**Pull request by omarASC5**
Added the ability to see yourself while recording a video submission for Truth Loop with @carrir2 https://github.com/carrir2.
Fixes #101

### CODE REVIEW

**Comment by krook** [comment]
@markstur or others, can you please review?

File: client/src/views/Record.vue [hunk 1]
```diff
@@ -1,95 +1,135 @@
...
+                    <cv-column class="col-2">
+                        <div class="btn-blur">Blur</div>
+                    </cv-column>
+                    <cv-column class="col-3">
+                        <div>Audio Only</div>
+                    </cv-column>
+                  </cv-row>
+                  <cv-row class="grid-row">
+                    <cv-column class="col-1">
+                      Lulu Blake
```
Comment (markstur) [inline client/src/views/Record.vue:50]: Is the hard-coded "Lulu Blake" here just a placeholder?

**Review by markstur — CHANGES_REQUESTED** [review]
One question about the hard-coded name.  If that okay, I'll merge it.

See inline comment

File: client/src/views/Record.vue (hunk 1 above)
Comment (omarASC5) [reply client/src/views/Record.vue:50]: Hi, that was already there, I was wondering the same thing. So, we just left it how it was.

Comment (markstur) [reply client/src/views/Record.vue:50]: thanks for the quick reply.  I'll merge this

**Review by markstur — APPROVED** [review]
Looks good.  Thanks!

**Comment by demilolu** [comment]
@omarASC5 it was great having you engaged with Truth Loop! We've kicked off some more work here and I wanted to know if you'd be interested in getting involved again. (I'm the new OS manager for Call for Code for Racial Justice).

**Comment by omarASC5** [comment]
@demilolu Sure, I have a pretty busy semester but I'll see if I can work on some issues during free time.

**Pull request merged**